test/e2e/persistent_volumes-disruptive.go

```diff
@@ -0,0 +1,263 @@
+
+	result, err = framework.SSH("mount", nodeIP, framework.TestContext.Provider)
+	Expect(err).NotTo(HaveOccurred())
+	if strings.Contains(result.Stdout, string(clientPod.UID)) {
+		framework.Failf("Client UID %s found in `mount` output.  Volume failed to unmount.", clientPod.UID)
+	} else {
+		framework.Logf("Client UID %s not found in `mount` output.  Volume has unmounted.", clientPod.UID)
+	}
+}
+
+// Initialize a test spec with a pv, pvc, and nfs client pod
+func initTestCase(f *framework.Framework, c clientset.Interface, pvConfig persistentVolumeConfig, ns, nodeName string) (pod *v1.Pod, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) {
+	pv, pvc = createPVPVC(c, pvConfig, ns, false)
+	pod = makePod(ns, pvc.Name)
+	pod.Spec.NodeName = nodeName
+	framework.Logf("Creating nfs client Pod %s on node %s", pod.Name, nodeName)
+	pod, err := c.Core().Pods(ns).Create(pod)
+	Expect(err).NotTo(HaveOccurred())
+	err = framework.WaitForPodRunningInNamespace(c, pod)
+	Expect(err).NotTo(HaveOccurred())
+
+	pod, err = c.Core().Pods(ns).Get(pod.Name, metav1.GetOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	pvc, err = c.Core().PersistentVolumeClaims(ns).Get(pvc.Name, metav1.GetOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	pv, err = c.Core().PersistentVolumes().Get(pv.Name, metav1.GetOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	return
+}
+
+// Post-spec clean up
+func tearDownTestCase(c clientset.Interface, f *framework.Framework, ns string, pod *v1.Pod, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) {
+	deletePod(f, c, ns, pod)
+	deletePersistentVolumeClaim(c, pvc.Name, ns)
+	deletePersistentVolume(c, pv.Name)
+}
+
+// Start, Restart, or Stop the kubelet running on the node of the target pod.
+// Allowed kubeltOps are `kStart`, `kStop`, and `kRestart`
+func kubeletCommand(kOp kubeletOpt, c clientset.Interface, pod *v1.Pod) {
+	nodeIP, err := framework.GetHostExternalAddress(c, pod)
+	Expect(err).NotTo(HaveOccurred())
+	nodeIP = nodeIP + ":22"
+	sshResult, err := framework.SSH("/etc/init.d/kubelet "+string(kOp), nodeIP, framework.TestContext.Provider)
+	Expect(err).NotTo(HaveOccurred())
+	framework.LogSSHResult(sshResult)
+
+	// On restart, waiting for node NotReady prevents a race condition where the node takes a few moments to leave the
+	// Ready state which in turn short circuits WaitForNodeToBeReady()
+	if kOp == kStop || kOp == kRestart {
+		if ok := framework.WaitForNodeToBeNotReady(c, pod.Spec.NodeName, NODE_STATE_WAIT); !ok {
+			framework.Failf("Node %s failed to enter NotReady state", pod.Spec.NodeName)
+		}
+	}
+	if kOp == kStart || kOp == kRestart {
+		if ok := framework.WaitForNodeToBeReady(c, pod.Spec.NodeName, NODE_STATE_WAIT); !ok {
+			framework.Failf("Node %s failed to enter NotReady state", pod.Spec.NodeName)
+		}
+	}
+
+}
+
+// Wraps RunKubectl to execute a bash cmd in target pod
+func podExec(pod *v1.Pod, bashExec string) (string, error) {
+	args := strings.Split(bashExec, " ")
```

Comment:
Just to keep the thread up w/ our offline convo:
`framework.RunKubectl(cmd...)` fails w/ 
>       Command stdout:
>        rpc error: code = 2 desc = "oci runtime error: exec failed: exec: \"touch /mnt/_SUCCESS\": stat touch /mnt/_SUCCESS: no such file or directory"
 
when the bash command string isn't broken into separate strings.